app/mailers/admin_mailer.rb

```diff
@@ -71,7 +71,11 @@ def notify_moderators_of_comment_spam(comment, moderator)
     @moderator = moderator
     @comment = comment
     @footer = feature('email-footer')
-    moderators = User.where(role: %w(moderator admin)).collect(&:email)
+    all_moderators = User.where(role: %w(moderator admin))
+    moderators = []
+    all_moderators.each do |mod_user|
+      moderators << mod_user.email unless mod_user.has_tag('no-moderation-emails')
```

Comment:
Ah, `notifications:noemail` tag was a proposed new tag, so none will exist yet. But if we could add that too, that'd be great just as a more general case than only for moderators. Could we include it with an `||` OR conditional?

Reply:
I have added tag `notifications:noemail` with or condition